src/commands/cont.rs

```diff
@@ -9,8 +9,50 @@ pub struct ContinueCommand;
 
 impl ContinueCommand {
     pub async fn execute(api_client: impl ApiClient) -> ResultWithDefaultError<()> {
+        let running_entry_stop_time = Utc::now();
+        let running_time_entry = api_client.get_running_time_entry().await;
+        match &running_time_entry {
+            Err(error) => {
+                println!(
+                    "{} {:?}",
+                    "Couldn't retrieve running time entry.".red(),
+                    error
+                );
+                return Ok(());
+            }
+            Ok(value) => {
```

Comment:
After #23 we will have nicely formatted API errors, so you can safely bubble up this error by unwrapping and let main tell the user there's a connection issue

```suggestion
        let running_time_entry = api_client.get_running_time_entry().await?;
```

Reply:
Nice! will simplify